### ISSUE TITLE
Performance degrades

### ISSUE THREAD

**Issue by YangJae96** [chenhao2345/ICE#8]
Why does the performance degrade when I use only 1 GPU with all the same hyper-parameter?

How should I change the parameters when using 1 GPU.

**Comment by chenhao2345** [comment]
The gpu number, batchsize and learning rate can always affect results, because we use DataParellel mode.

I used 4 GPUs to train. Thus, the batchsize and learning rate values in our paper are tuned based on 4 GPUs.

If you have enough GPUs, try to use 4 GPUs. Otherwise, you may need to search optimal batchsize and learning rate values to find best results. Wish this can help you.
